src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  it("renders correctly", () => {
+    const results = [];
+    const { asFragment } = render(<SearchResults results={results} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders correctly with results", () => {
+    const results = [
+      "https:/example.com/image1.jpg",
+      "https:/example.com/image2.jpg",
+      "https:/example.com/image3.jpg",
+    ];
+
+    const { asFragment } = render(<SearchResults results={results} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
You cant render more than one snapshot in a test file. Since you already have a test below to assert the correct number of images are being rendered this should just be deleted